ttnn/cpp/pybind11/__init__.cpp

```diff
@@ -2,16 +2,16 @@
 //
 // SPDX-License-Identifier: Apache-2.0
 
+#include "operations/__init__.hpp"
```

Comment:
Here is some sorting going on. We should consider to disable sorting

Reply:
`SortIncludes:    true`

Reply:
I personally like the sorting because if you have problems with the includes it's a good way to to notice them.

Reply:
Agree, but should not be enabled by default before all related issues are resolved. Paul shared that there are some.